profiles/thurloe/terraform/thurloe/thurloe.tf

```diff
@@ -1,92 +1,115 @@
 
 # Cloud SQL database
-module "terra-cloudsql" {
-  source        = "github.com/broadinstitute/terraform-shared.git//terraform-modules/cloudsql-mysql?ref=rl-add-modules"
+module "cloudsql" {
+  source        = "github.com/broadinstitute/terraform-shared.git//terraform-modules/cloudsql-mysql"
```

Comment:
This should still refer to a specific tag (if one doesn't exist already you can make one in the format `cloudsql-mysql-0.0.1`) so that it doesn't break when we update the module

Reply:
Added